SPECS/protobuf/protobuf.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Google's data interchange format
 Name:           protobuf
-Version:        3.17.3
+Version:        3.18.3
```

Comment:
We try to avoid version changes to something higher than the patch version (the third number in the version). This is to avoid breaking changes as much as possible. The changes in the `%files` section indicate that we'd need to re-compile all binaries, which built against the older version and that's tricky, since we don't know where all of these binaries might have built (we're aware about our packages, but not external projects).

With that in mind, can you try to port the CVE patch to the 3.17.X version?

Reply:
I have looked into getting a patch at first. But unfortunately the only patch is to upgrade to certain *versions*, they did not specifically produce the patch for it as it is parsing and memory management issue that affects specific versions of protobuf